api/src/main/java/org/openmrs/module/idgen/AutoGenerationOption.java

```diff
@@ -22,7 +22,6 @@
 public class AutoGenerationOption {
 	
 	//***** PROPERTIES *****
-	
```

Comment:
why did you change this

Reply:
I didn't. It was like that when I rebased.

Reply:
It shows as a change from your end

Reply:
If you didn't change them then your commits are mixed with someone else's commits

Reply:
remove this file from your commit

Reply:
Your commit should not have any changes to this file.